v21.2/architecture/storage-layer.md

```diff
@@ -51,6 +51,80 @@ Pebble integrates well with CockroachDB for a number of reasons:
 
 Efficient storage for the keys is guaranteed by the underlying Pebble engine by means of prefix compression.
 
+For more information about Pebble, see the [Pebble GitHub page](https://github.com/cockroachdb/pebble) or the blog post [Introducing Pebble: A RocksDB Inspired Key-Value Store Written in Go](https://www.cockroachlabs.com/blog/pebble-rocksdb-kv-store/).
+
+Pebble uses a log structured merge tree (_LSM_) to manage data storage. For more information about how LSM-based storage engines like Pebble work, see [Log structured merge trees](#log-structured-merge-trees) below.
+
+#### Log structured merge trees
+
+Pebble uses a Log structured merge tree (hereafter _LSM tree_ or _LSM_) to manage data storage. The LSM is a hierarchical tree. At each level of the tree, there are files on disk that store the data referenced at that level. The files are known as _sorted string table_ files (hereafter _SST_ or _SST file_).
+
+##### SSTs
+
+SSTs are an on-disk representation of sorted lists of key-value pairs. Conceptually, they look something like this:
```

Comment:
This is an appropriate simplification, but we could explicitly say it's a simplification as it skips some things (eg. index blocks).

Reply:
I updated to say "they look something like this (intentionally simplified) diagram" - do you think that makes it clearer?

Reply:
Yep, that sounds good!